web/app/css/deployment.css

```diff
@@ -5,6 +5,33 @@
     margin-bottom: 30px;
   }
 
+  & .deployment-title {
+    margin: 6px 0;
+
+    & h1 {
+      display: inline-block;
+    }
+  }
+
+
+
+  & .status-badge {
+    display: inline-block;
+    border-radius: 50px;
+    padding: 4px 10px;
+    margin: 6px 12px;
+    font-size: small;
+
+    &.added {
+      background: green;
+    }
+
+    &.unadded {
+      background: #BDBDBD;
```

Comment:
probably we should also put this in a variable, as we're adding it in more and more places